project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.3
+sbt.version=1.3.13
```

Comment:
This SBT bump is because the following publishing job failed to download assets for SBT 1.3.3:
https://jenkins.mesosphere.com/service/jenkins/blue/organizations/jenkins/usi-publishing/detail/PR-160/2/pipeline